README.md

```
@@ -1 +1,26 @@
+
+- [x] 모달을 열 수 있어야 한다
+- [x] 모달을 닫을 수 있어야 한다
+  - [x] x버튼을 클릭했을 때
+  - [x] backDrop 클릭했을 때
+  - [x] esc 버튼을 눌렀을 때
+- [ ] props를 통해서 모달을 구성할 수 있다
+  - [ ] 모달 위치 : 중앙, 하단
+  - [x] 모달 제목
+  - [x] 모달 콘텐츠
+
+**페이먼츠 커스텀 훅**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)